src/client/menu/videomenu.c

```diff
@@ -499,33 +508,58 @@ VID_MenuInit(void)
 	if (strcmp(vid_renderer->string, "gl3") == 0 || strcmp(vid_renderer->string, "gles3") == 0)
 	{
-		s_intensity_slider.generic.type = MTYPE_SLIDER;
-		s_intensity_slider.generic.name = "color intensity";
-		s_intensity_slider.generic.x = 0;
-		s_intensity_slider.generic.y = (y += 10);
-		s_intensity_slider.cvar = "gl3_intensity";
-		s_intensity_slider.minvalue = 0.1f;
-		s_intensity_slider.maxvalue = 256.0f;
-
-		s_overbrightbits_slider.generic.type = MTYPE_SLIDER;
-		s_overbrightbits_slider.generic.name = "overbrights";
-		s_overbrightbits_slider.generic.x = 0;
-		s_overbrightbits_slider.generic.y = (y += 10);
-		s_overbrightbits_slider.cvar = "gl3_overbrightbits";
-		s_overbrightbits_slider.minvalue = 0.1f;
-		s_overbrightbits_slider.maxvalue = 256.0f;
+		gl1_colorlight = 0;
+		s_gl3_intensity_slider.generic.type = MTYPE_SLIDER;
+		s_gl3_intensity_slider.generic.name = "color intensity";
+		s_gl3_intensity_slider.generic.x = 0;
+		s_gl3_intensity_slider.generic.y = (y += 10);
+		s_gl3_intensity_slider.cvar = "gl3_intensity";
+		s_gl3_intensity_slider.minvalue = 0.1f;
+		s_gl3_intensity_slider.maxvalue = 256.0f;
+
+		s_gl3_overbrightbits_slider.generic.type = MTYPE_SLIDER;
+		s_gl3_overbrightbits_slider.generic.name = "overbrights";
+		s_gl3_overbrightbits_slider.generic.x = 0;
+		s_gl3_overbrightbits_slider.generic.y = (y += 10);
+		s_gl3_overbrightbits_slider.cvar = "gl3_overbrightbits";
+		s_gl3_overbrightbits_slider.minvalue = 0.1f;
+		s_gl3_overbrightbits_slider.maxvalue = 256.0f;
 
 		gl3_colorlight = Cvar_Get("gl3_colorlight", "1", CVAR_ARCHIVE);
-		s_colorlight_list.generic.type = MTYPE_SPINCONTROL;
-		s_colorlight_list.generic.name = "color light";
-		s_colorlight_list.generic.x = 0;
-		s_colorlight_list.generic.y = (y += 10);
-		s_colorlight_list.itemnames = yesno_names;
-		s_colorlight_list.curvalue = (gl3_colorlight->value != 0);
+		s_gl3_colorlight_list.generic.type = MTYPE_SPINCONTROL;
+		s_gl3_colorlight_list.generic.name = "color light";
+		s_gl3_colorlight_list.generic.x = 0;
+		s_gl3_colorlight_list.generic.y = (y += 10);
+		s_gl3_colorlight_list.itemnames = yesno_names;
+		s_gl3_colorlight_list.curvalue = (gl3_colorlight->value != 0);
 	}
 	else
 	{
 		gl3_colorlight = 0;
+		s_gl1_intensity_slider.generic.type = MTYPE_SLIDER;
+		s_gl1_intensity_slider.generic.name = "color intensity";
+		s_gl1_intensity_slider.generic.x = 0;
+		s_gl1_intensity_slider.generic.y = (y += 10);
+		s_gl1_intensity_slider.cvar = "gl1_intensity";
+		s_gl1_intensity_slider.minvalue = 1.0f;
+		s_gl1_intensity_slider.maxvalue = 10.0f;
+
+		s_gl1_overbrightbits_slider.generic.type = MTYPE_SLIDER;
+		s_gl1_overbrightbits_slider.generic.name = "overbrights";
+		s_gl1_overbrightbits_slider.generic.x = 0;
+		s_gl1_overbrightbits_slider.generic.y = (y += 10);
+		s_gl1_overbrightbits_slider.cvar = "gl1_overbrightbits";
+		s_gl1_overbrightbits_slider.minvalue = 0;
+		s_gl1_overbrightbits_slider.maxvalue = 3;
+		s_gl1_overbrightbits_slider.printformat = "%.0f";
```

Comment:
Enforcing a printformat / enforcing it this way messes up the scale. Just give `minvalue` and `maxvalue` as floats, remove the `printformat` and it should be fine.

Reply:
> Enforcing a printformat / enforcing it this way messes up the scale. Just give `minvalue` and `maxvalue` as floats, remove the `printformat` and it should be fine.

Better than that, he should give `slidestep` a value of 1, since gl_overbrightbits is an integer value, and it should change in steps of '1' instead of the default of  '0.1'. See the FOV slider for a fully working example.

Reply:
Ah yes, you are right. Of course.